platform_versioned_docs/version-23.4/getting-started/rnaseq.mdx

```diff
@@ -0,0 +1,752 @@
+  - A **Description** for the dataset.
+  - Select the **First row as header** option to prevent Platform from parsing the header row of the samplesheet as sample data.
+  - Select **Upload file** and browse to your CSV or TSV samplesheet file in local storage, or simply drag and drop it into the box.
+
+  The dataset is now listed in your organization workspace datasets and can be selected as input when launching your pipeline. 
+
+  :::info
+  Platform does not store the data used for analysis in pipelines. The dataset must specify the locations of data stored on your own infrastructure.
+  :::
+
+</details>
+
+## Launch pipeline
+
+With your compute environment created, nf-core/rnaseq added to your workspace Launchpad, and your samplesheet accessible in Platform, you are ready to launch your pipeline. Navigate to the Launchpad and select **Launch** next to `nf-core-rnaseq` to open the launch form.
+
+ ![Launch a pipeline](./_images/launch-form-1.gif)
+
+The launch form consists of **General config**, **Run parameters**, and **Advanced options** sections to specify your run parameters before execution, and an execution summary. Use section headings or select the **Previous** and **Next** buttons at the bottom of the page to navigate between sections. 
+
+### General config 
+
+![General config tab](./_images/launch-form-2.gif)
+
+- **Pipeline to launch**: The pipeline Git repository name or URL. For saved pipelines, this is prefilled and cannot be edited.
+- **Revision number**: A valid repository commit ID, tag, or branch name. For saved pipelines, this is prefilled and cannot be edited.
+- **Config profiles**: One or more [configuration profile](https://www.nextflow.io/docs/latest/config.html#config-profiles) names to use for the execution. Config profiles must be defined in the `nextflow.config` file in the pipeline repository.
+- **Workflow run name**: An identifier for the run, pre-filled with a random name. This can be customized.
+- **Labels**: Assign new or existing [labels](../labels/overview.mdx) to the run.
+- **Compute environment**: Your AWS Batch compute environment. 
+- **Work directory**: The cloud storage path where pipeline scratch data is stored. Platform will create a scratch sub-folder if only a cloud bucket location is specified.
+  :::note
+  The credentials associated with the compute environment must have access to the work directory.
+  :::
+
+### Run parameters 
+
+![Run parameters](./_images/launch-form-3.gif)
+
+There are three ways to enter **Run parameters** prior to launch:
+
+- The **Input form view** displays form fields to enter text, select attributes from dropdowns, and browse input and output locations with [Data Explorer](../data/data-explorer.mdx).
+- The **Config view** displays raw configuration text that you can edit directly. Select JSON or YAML format from the **View as** dropdown.
+- **Upload params file** allows you to upload a JSON or YAML file with run parameters.
```

Comment:
I'm not sure if there is a Seqera way of writing dropdown

Reply:
We have no set rule for this, and Google style guide recommends dropping the term altogether unless omitting it would be confusing. So, I'm stripping the term drop-down out everywhere on the page. Thanks.